Task_10/index.html

```diff
@@ -0,0 +1,143 @@
+        <div class="lorem_collum" > Lorem ipsum dolor sit amet consectetur adipisicing elit. Quidem eaque quibusdam, at sapiente ratione ut iste rem animi recusandae dolore quod facilis ab non earum aperiam aliquid. Tenetur, aut eius.<br> <span class="col_1" >read more...</span> </div>
+        <div class="lorem_collum" >Lorem ipsum, dolor sit amet consectetur adipisicing elit. Repellat officia nam odit minus nulla repellendus porro corporis impedit! Earum sed pariatur dolor quas repudiandae dolores iure soluta rerum, ratione quis.<br> <span class="col_1" >read more...</span></div>
+        <div class="lorem_collum" > Lorem ipsum dolor sit amet consectetur adipisicing elit. Odit at, obcaecati eius perspiciatis eum beatae officiis veniam quasi esse repudiandae ipsam velit consequuntur, iure ab, excepturi itaque! Quod, rerum eius.<br> <span class="col_1" >read more...</span></div>
+        </div>
+    <!-- ///////////////////////////////// -->
+    <div>
+        <div class="meet">Meet the team</div>
+
+    </div>
+       
+   <div class="flex_image_rabbit">
+    
+
+        <div class="rabbit_img">
+        <div class="figure_rabbit">
+            <div class="span_center" ><span class="black_text">Hoof</span>-<span  class="orange_text">Lead Designer</span></div>
+            <div class="rabbit1"></div>
+            <div>Lorem, ipsum dolor sit amet consectetur adipisicing elit. Ea id aliquam facere ex, voluptatibus deleniti? Laudant</div>
+                </div>
+
+                <div class="figure_rabbit">
+                    <div class="span_center"> <span class="black_text">Nids</span>-<span class="orange_text">CEO/Director</span></div>
+                    <div class="rabbit2"></div>
+                    <div>Lorem, ipsum dolor sit amet consectetur adipisicing elit. Ea id aliquam facere ex, voluptatibus deleniti? Laudant</div>
+                        </div>
+                 <div class="figure_rabbit" >
+                            <div class="span_center" ><span class="black_text">Squawk</span>-<span class="orange_text">Developer</span></div>
+                            <div class="rabbit3"></div>
+                            <div>Lorem, ipsum dolor sit amet consectetur adipisicing elit. Ea id aliquam facere ex, voluptatibus deleniti?</div>
+                  </div>
+                </div> 
+            
+
+                <!-- ///////////////////////////////// -->
+                <div class="text_input1">Send us a message...</div>
+                
+
+                
+        <div class="container_input">
+            <form>
+              <div class="flex-outer">
+                  <div>
+                  <label> Name:  </label>
```

Comment:
Label and textarea don't connect with each other

Reply:
seems to have fixed